docs/data_ctmeter.md

```
@@ -1,17 +1,16 @@
-- [Envoy.data.ctmeter_storage](pyenphase.EnvoyData.ctmeter_storage)
-- [Envoy.data.ctmeter_production_phases](pyenphase.EnvoyData.ctmeter_production_phases)
-- [Envoy.data.ctmeter_consumption_phases](pyenphase.EnvoyData.ctmeter_consumption_phases)
-- [Envoy.data.ctmeter_storage_phases](pyenphase.EnvoyData.ctmeter_storage_phases)
+- {py:attr}`pyenphase.EnvoyData.ctmeter_production`
+- {py:attr}`pyenphase.EnvoyData.ctmeter_consumption`
+- {py:attr}`pyenphase.EnvoyData.ctmeter_storage`
 
-There is one ct for production, one for consumption and one for battery storage. One, 2 or all of these can be installed and enabled. Data is only available if CT's are enabled. Consumption CT can either be `net-consumption` installed on grid boundary or `total-consumption` measuring house load, see [ct-model](#ct-model) below.
+There is multiple types of CT that can be installed. The CT meter types are enumerated as `production`, `storage`, `net-consumption` and `total-consumption` by `pyenphase.models.meters.CtType`. One, or multiple of these can be installed and enabled. For multi-phase configurations there will be one on each phase.
+
+Consumption CT can either be `net-consumption` installed on grid boundary or `total-consumption` measuring house load, see [ct-model](#) below. The IQ Metered collar has a `net-consumption` CT embedded.
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)